scripts.d/support/install-setup-common

```diff
@@ -115,6 +119,27 @@ function template-conversion()
             done
     fi
 
+    # cfg file from CFD with paths to patch
+
+    local FILES_CFG=(*/lib/code_saturne_build.cfg)
+    local TOOLS_LIST=( s-saturne s-libccmio s-eos s-coolprop s-saturne-dbg )
```

Comment:
We can (should) not based these changes on a list of modules. If there is more in the future this will be broken.

We really need to parse the initial string using section and replace with `\1` ... to replace all pattern.